client/src/views/Simulation/components/ModelPanel.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+  <section class="simulation-model-panel">
+    <settings-bar>
+      <counters slot="left" :title="modelName" :data="countersData" />
+      <aside slot="right">
+        <div class="dropdown">
+          <button
+            class="btn btn-primary dropdown-toggle"
```

Comment:
I think we should be using `btn-secondary` for these settings buttons as well as for the expand panel button. In this way, we highlight buttons such as `run`, `provenance graph`, etc a bit more.

Reply:
![image](https://user-images.githubusercontent.com/10552785/122446052-39539d80-cf70-11eb-9ad0-a4922182c24f.png)